src/pages/NavbarPage.tsx

```diff
@@ -0,0 +1,131 @@
+import { CodeExample, DemoPage } from './DemoPage';
+import { Navbar } from '../components';
+import { Button } from '../components';
+
+const NavbarPage: FC = () => {
+  const examples: CodeExample[] = [
+    {
+      title: 'Default Navbar',
+      code: (
+        <Navbar>
+          <Navbar.Brand href="https://flowbite.com/">
+            <img src="https://flowbite.com/docs/images/logo.svg" className="mr-3 h-6 sm:h-9" alt="Flowbite Logo" />
+            <span className="self-center whitespace-nowrap text-xl font-semibold dark:text-white">Flowbite</span>
+          </Navbar.Brand>
+          <Navbar.Toggle />
+          <Navbar.Collapse>
+            <ul className="mt-4 flex flex-col md:mt-0 md:flex-row md:space-x-8 md:text-sm md:font-medium">
+              <li>
+                <a
+                  href="#/navbars"
+                  className="block rounded bg-blue-700 py-2 pr-4 pl-3 text-white dark:text-white md:bg-transparent md:p-0 md:text-blue-700"
+                  aria-current="page"
+                >
+                  Home
+                </a>
+              </li>
+              <li>
+                <a
+                  href="#/navbars"
+                  className="block border-b border-gray-100 py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
+                >
+                  About
+                </a>
+              </li>
+              <li>
+                <a
+                  href="#/navbars"
+                  className="block border-b border-gray-100 py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
+                >
+                  Services
+                </a>
+              </li>
+              <li>
+                <a
+                  href="#/navbars"
+                  className="block border-b border-gray-100 py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
+                >
+                  Pricing
+                </a>
+              </li>
+              <li>
+                <a
+                  href="#/navbars"
+                  className="block py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
+                >
+                  Contact
+                </a>
+              </li>
+            </ul>
```

Comment:
I want to use something like `Navbar.Item` or `Navbar.Link` with active attrs to simplify this a bit. My method is similar to React-Bootstrap's design pattern but would love feedback on this.